lighthouse-core/config/index.js

```diff
@@ -171,16 +172,79 @@ function filterAudits(audits, auditWhitelist) {
   return filteredAudits;
 }
 
-function expandAudits(audits) {
+function expandAudits(audits, paths) {
+  const rootPath = path.join(__dirname, '../../');
+
   return audits.map(audit => {
-    try {
-      return require(`../audits/${audit}`);
-    } catch (requireError) {
+    // Check each path to see if the audit can be located. First match wins.
+    const AuditClass = paths.reduce((definition, auditPath) => {
+      // If the definition has already been found, just propagate it. Otherwise try a search
+      // on the path in this iteration of the loop.
+      if (definition !== null) {
+        return definition;
+      }
+
+      const requirePath = auditPath.startsWith('/') ? auditPath : path.join(rootPath, auditPath);
```

Comment:
Yup: The last 3 here are exactly what we need:
![image](https://cloud.githubusercontent.com/assets/39191/17722617/57f16e78-63e8-11e6-8046-dd84dbd2c983.png)